### ISSUE TITLE
Odd repetition and other issues recently - Incorrect ftype?

### ISSUE THREAD

**Issue by icsy7867** [LostRuins/koboldcpp#1192]
**Describe the Issue**
Apologies, I am in no means an expert, and I am still learning.

Recently, after upgrading KoboldCPP I have been seeing some strange repetition and other issues with responses that frequently just dont quite make sense given the content.

I am well within the context limit.  (Currently testing Qwen 2.5 - 32B using Q4 quantization), it the responses frequently contradict what is in the context right before it.  I was doing some digging, so I noticed that in the start output I see:

```
llm_load_print_meta: model ftype      = Q3_K - Large
```

even though this is a Q4 model.  Being curious, I downloaded llamacpp and did a similar  execution, and it immediately detected the correct Q4 on the same model.

**Additional Information:**
Running in Podman, using a Quadro P6000, 24GB VRAM.

For running koboldcpp I am using:

```
--usecublas --flashattention --gpulayers 999 --contextsize 12000
```

Seems to work otherwise, no errors or anything visible.  Just curious if someone new of a magic flag or something to try?  Or maybe I am being stupid and missing something.

**Comment by icsy7867** [comment]
I also test version 1.74 and version 1.73, but it always identifies the ftype as Q3_Large.

**Comment by LostRuins** [comment]
I don't think the ftype detection has anything to do with the output quality - if it was wrong it would output complete garbage.

**Comment by icsy7867** [comment]
Fair enough! Thank you for the response.  I am still exploring the various flags and options as I figure out the issues.  I am pretty sure this is user error somehow...
